### ISSUE TITLE
Query-able database for Ethereum

### ISSUE THREAD

**Issue by shankars99** [Web3-Conf-India/Proposal-2022#200]
What? Querying? Like querying for data? 
Yes, you got that right. 

Data is a problem on Ethereum. You can't store a lot of data on smart contracts the same way you can do on a regular server based application. 

Why? Because you would be honestly, quite very poor at the end of it and everyone using the dapp would too.
Storing data costs money, the more data the more money, a simple relationship since the historic times of when people used google drive, and still do.

Why would you want to store data on a blockchain in massive quantities? 
Maybe you want decentralization, data storage with mathematical proofs, or any other reason why you love blockchains.

So if we can't store data on chain, how do do solve this problem?
Just send it off-chain. Thanks for attending my ted talk have a great day.

Except we're not done. 

StorkNet is a l2 queryable data-layer, that I created.
tl;dr it's mongoDB for Ethereum. 

Don't worry I'm not here to advertise it, I'm here to explain the thought process behind it.

Now you can decouple data from the chain, and make it stateless, as you're not storing on-chain you're paying much less gas, reducing the size of the Ethereum mainnet, and a lot of other things, also including cross-chain data transfers.

In the discussion we'll be talking about 
1. Storage problems on Ethereum mainnet
2. How to tackle them 
3. The idea behind StorkNet
4. The different ways you can do the same
5. When you'd want to do something like this
